db/fixtures/notification_types.yml

```diff
@@ -179,6 +179,11 @@
   :expires_in: 24.hours
   :level: :error
   :audience: global
+- :name: vm_resize_error
+  :message: '%{message}'
```

Comment:
Typically this has a "static" string component which gets translated, then just the error message gets interpolated into that.

How about `:message: 'Resize VM %{subject} failed: %{error}'`

Then https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/300/files#diff-d2062f3ae70afc4bd9041f6d6362e5005169c3e251ab50b5cb85957cf5a93e1bR84 becomes
`Notification.create(:type => :vm_resize_error, :options => {:subject => name, :error => e.error}`